services/basic_types.proto

```diff
@@ -264,6 +265,12 @@ message TokenTransferList {
      * which has a sender and receiver account, including the serial number of the NFT
      */
     repeated NftTransfer nftTransfers = 3;
+
+    /**
+     * If present, the number of decimals this fungible token type is expected to have. The transfer
+     * will fail with UNEXPECTED_TOKEN_DECIMALS if the actual decimals differ.
+     */
+    google.protobuf.Int32Value expected_decimals = 4;
```

Comment:
Even though I usually prefer signed since Java doesn't have unsigned, I'm wondering if this should be `UInt32Value` to match the `TokenCreate.decimals` being `uint32`?